persistence-modules/hibernate5/src/test/java/com/baeldung/hibernate/persistjson/PersistJSONLiveTest.java

```diff
@@ -0,0 +1,111 @@
+package com.baeldung.hibernate.persistjson;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+import org.hibernate.HibernateException;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.MetadataSources;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.cfg.Configuration;
+import org.hibernate.service.ServiceRegistry;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class PersistJSONLiveTest {
```

Comment:
Since we are just using H2, I'm guessing that this could actually run with the build, right? Might consider making this a UnitTest so that it gets run by Travis.